scripts/build-subprojects.sh

```diff
@@ -13,9 +13,12 @@ case "$SUBPROJECT" in "rosette")
 
     sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate casper/test:compile coverage scalafmtCheck test coverageReport rspace:tut
 
+    curl -s https://codecov.io/bash >/tmp/codecov
+    chmod +x /tmp/codecov
```

Comment:
@adaszko You're definitely right that we should be cautious about running arbitrary code from Internet. However, this only runs on Travis where in every build we (have to?) setup whole environment from Travis predefined bare Ubuntu distribution. Same argument could be made for SBT/Ivy dependencies and we definitely don't want to bundle these in repository (some of them are plugins that can run arbitrary code during compilation). We can at least rely on TLS (https://) to prevent MITM code injections.